migrations/1564534118044_create-deployments.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Deployment } from '../src/Deployment'
+
+const tableName = Deployment.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      user_id: { type: 'TEXT', notNull: true },
+      lastPublishedCID: { type: 'TEXT' },
+      isDirty: { type: 'BOOLEAN', notNull: true },
```

Comment:
should we name `lastPublishedCID` and `isDirty` also in snake_case? We are already doing a transformation from the response in the frontend so we could transform those too and keep the naming consistent here.

Reply:
Indeed! I missed those two and used the old test deployment so I didn't realised.

PR Updated, both the code and Postman test files